.circleci/config.yml

```diff
@@ -7,4 +7,4 @@ jobs:
       - checkout
       - run: bundle install
       - run: bundle exec danger --verbose
-      - run: bundle exec awesome_bot --allow-redirect --allow-dupe -w awesome_bot --allow-redirect --allow-dupe -w https://github.com/Shift3/ifg/compare/v0.4.0...v0.5.0,https://github.com/Shift3/sinclair-scanner-application,https://github.com/Shift3/sinclair-scanner-application/pull/137,https://github.com/Shift3/FUSDMonitoringAppXamarin,https://github.com/Shift3/rn-by-example,https://github.com/Shift3/fusd-front-desk-kiosk,regexr,https://github.com/Shift3/bw-project-status,https://github.com/Shift3/Aerostar,https://github.com/Shift3/qa-team/projects,https://github.com/Shift3/sundial-mobile-app,127.0.0.1 `find . -name "*.md"`
+      - run: bundle exec awesome_bot --allow-redirect --allow-dupe -w awesome_bot --allow-redirect --allow-dupe -w https://github.com/Shift3/ifg/compare/v0.4.0...v0.5.0,https://github.com/Shift3/sinclair-scanner-application,https://github.com/Shift3/sinclair-scanner-application/pull/137,https://github.com/Shift3/FUSDMonitoringAppXamarin,https://github.com/Shift3/rn-by-example,https://github.com/Shift3/fusd-front-desk-kiosk,regexr,https://github.com/Shift3/bw-project-status,https://github.com/Shift3/Aerostar,https://github.com/Shift3/qa-team/projects,https://github.com/Shift3/sundial-mobile-app,https://github.com/Shift3/boilerplate-client-angular#docker,https://github.com/Shift3/boilerplate-server-node/blob/develop/docker-compose.yml,127.0.0.1 `find . -name "*.md"`
```

Comment:
It would be helpful to explain the concept of this command.

Reply:
Awesome_bot checks all links within READMEs recursively to make sure they are still valid. The edit comes from a series of Bitwise/Shift3 specific links that cannot be checked as they are private repos. 

Im thinking of removing awesome_bot entirely from individual PRs like this one and changing it to a semi-weekly cron job kind of setup within circleci but its still in the works. For now, this has proven to work on keeping links up to date.

Reply:
Oh super cool. good to know about it. Thanks for clarifying that for me.